### PR TITLE
Add pan-zoom feature for Image canvases.

### PR DESCRIPTION
**Pull request by mathewjordan**
## What does this do?
This pull request brings in the OpenSeadragon viewer into the RMP package for viewing of Image based canvases. Functionality for this is limited to zoom in, zoom out and reset of the viewer. All controls are entirely keyboard operable. Upon switching canvases, the item should render centered in the OpenSeadragon canvas. The instance will remain intact while toggling image canvases and will destroy itself when the user toggles to a non-Image canvas.

image

### CODE REVIEW

**Review by adamjarling — CHANGES_REQUESTED** [review]
Mind handling these in the console?

image

**Comment by adamjarling** [comment]
@mathewjordan Also on first load looks like I'm getting times for image file sets: 

image

File: package.json [hunk 1]
```diff
@@ -50,15 +50,18 @@
     "@radix-ui/react-radio-group": "^0.1.1",
     "@radix-ui/react-tabs": "^0.1.1",
     "@stitches/react": "^1.2.0",
-    "@types/jest": "^27.0.2",
```
[the remark this answers is not in the document]
Comment (mathewjordan) [reply package.json:53]: @adamjarling I was getting a console warning with it being in both `dependencies` and `devDependencies` so I removed it in this spot. I believe we can remove it here.

**Comment by mathewjordan** [comment]
> @mathewjordan Also on first load looks like I'm getting times for image file sets:
> 
> image

This has been there for a bit, want me to handle in this issue or a new one?

**Comment by mathewjordan** [comment]
> Mind handling these in the console?
> 
> image

These are pretty ugly but might be unrelated to the OpenSeadragon. I'm surprised I didn't see them previously.

**Comment by adamjarling** [comment]
Hey @mathewjordan Yea, I'd say for the showing the time for an Image, that should be pretty quick and feels to me like it's related enough to displaying images, so yea let's include that in this issue.

For the `play()` error, if that's unrelated we can create a new bug issue.   Your call.   

Are you seeing the CORS errors, or is that just on my end?   Running your branch I'm, as you can see, getting network errors pulling in the IIIF images.   Is this just me?

**Comment by mathewjordan** [comment]
@adamjarling 

> Hey @mathewjordan Yea, I'd say for the showing the time for an Image, that should be pretty quick and feels to me like it's related enough to displaying images, so yea let's include that in this issue.

👍 Handled

> For the `play()` error, if that's unrelated we can create a new bug issue. Your call.

Yeah, I think this should be handled separate. I believe it was brought in by [this line ](https://github.com/nulib/react-media-player/blob/main/src/components/Player/Player.tsx#L90) a few PRs ago. We should probably talk it through as it is causing the player to autoplay at times when browsers are not like it.

> Are you seeing the CORS errors, or is that just on my end? Running your branch I'm, as you can see, getting network errors pulling in the IIIF images. Is this just me?

I'm not getting the the 400s on the IIIF requests. I'm not sure why you would I wouldn't. Odd. Perhaps I should create a fixture that isn't referencing staging resources?

**Review by adamjarling — APPROVED** [review]
Yea, looks better now in the `console`.  I'm still not able to load any IIIF resources for some reason due to CORS, but hopefully that'll work itself out in the player in Meadow and DC.

**Comment by adamjarling** [comment]
With the CORS errors I can't really test this, but the screen shot looks tight.

**Pull request merged**